modules/openapi-generator/src/main/resources/kotlin-spring/apiController.mustache

```diff
@@ -13,13 +13,16 @@ import java.util.Optional
 {{/useRequestMappingOnController}}
 {{#operations}}
 class {{classname}}Controller(
-        @org.springframework.beans.factory.annotation.Autowired(required = false) delegate: {{classname}}Delegate?
+        {{#skipDefaultDelegateInterface}}private val delegate: {{classname}}Delegate{{/skipDefaultDelegateInterface}}
+        {{^skipDefaultDelegateInterface}}delegate: {{classname}}Delegate?{{/skipDefaultDelegateInterface}}
```

Comment:
please refactor to 
```
        {{#skipDefaultDelegateInterface}}
        private val delegate: {{classname}}Delegate
        {{/skipDefaultDelegateInterface}}
        {{^skipDefaultDelegateInterface}}
        delegate: {{classname}}Delegate?
        {{/skipDefaultDelegateInterface}}
```

to avoid empty line break

Reply:
fixed